saplings/profile/src/components/KeyTable.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import './KeyTable.scss';
+import { Link } from 'react-router-dom';
+import Icon from '@material-ui/core/Icon';
+
+export function KeyTable() {
+  return (
+      <section className="user-keys">
+        <div className='keys-header'>
+          <h3 id="keys-label">Keys</h3>
+        </div>
+        <div className="table-actions">
+          <div className="table-nav">
+            <div className="total-keys">2 Keys</div>
+            <div className="row-info">
+              <div className="rows-label">Rows per page:</div>
+              <div className="rows-per-page">10</div>
+            </div>
+            <div className="keys-currently-displayed">1-2 of 2</div>
+            <div className="paging">
+              <div className="page-nav"><Icon>keyboard_arrow_left_icon</Icon></div>
+              <div className="page-nav"><Icon>keyboard_arrow_right_icon</Icon></div>
+            </div>
+          </div>
+          <Link className="add-key" to="/profile/new-key">
+            <div className="icon"><Icon>add_icon</Icon></div>
+            New Key
+          </Link>
+        </div>
+        <div className="table-wrapper">
+        <table>
+          <tr>
```

Comment:
This should be wrapped in `<thead>` (A console error occurs due to this)

Reply:
fixed